README.md

```diff
@@ -93,6 +93,23 @@ export OPENSHIFT_PYTHON_WRAPPER_LOG_LEVEL=<LOG_LEVEL> # can be: "DEBUG", "INFO",
   export OPENSHIFT_PYTHON_WRAPPER_HASH_LOG_DATA="false"
   ```
 
+## Proxy Enablement
+
+This configuration allows the client to route traffic through a specified proxy server.
+It can be enabled via the environment variable `OPENSHIFT_PYTHON_WRAPPER_CLIENT_USE_PROXY`.
+
+To enable proxy configuration for the client:
+
+1. Set the environment variable `OPENSHIFT_PYTHON_WRAPPER_CLIENT_USE_PROXY=<any value>`
```

Comment:
Can you give example? What are the valid options? or is the goal to ensure that it is set?
e.g. Currently if I set it to `OPENSHIFT_PYTHON_WRAPPER_CLIENT_USE_PROXY=False`, the code would still set proxy provided HTTPS_PROXY or HTTP_PROXY exists.

Reply:
Any value is ok and makes sense

Reply:
@myakove Should that be mentioned that you can't disable proxy using OPENSHIFT_PYTHON_WRAPPER_CLIENT_USE_PROXY? One needs to unset this environment variable? Currently it is not clear.